src/pages/editor/EditorInstitution.tsx

```diff
@@ -91,6 +114,40 @@ export const EditorInstitution = () => {
             {customer?.rboInstitution ? t('editor.institution.rbo_funded') : t('editor.institution.not_rbo_funded')}
           </Typography>
 
+          {institutionUsers && (
+            <Box sx={{ display: 'flex', flexWrap: 'wrap', gap: '3rem' }}>
+              {institutionAdmins && (
+                <div>
+                  <Typography variant="h3" gutterBottom>
+                    {institutionAdmins.length > 0
+                      ? t('editor.institution.institution_admin_plural')
+                      : t('editor.institution.institution_admin_single')}
+                  </Typography>
+                  {institutionAdmins.length > 0 ? (
+                    institutionAdmins.map((admin) => <InstitutionUserLink key={admin.cristinId} user={admin} />)
+                  ) : (
+                    <Typography>{t('editor.institution.institution_has_no_administrator')}</Typography>
+                  )}
+                </div>
+              )}
+
+              {institutionEditors && (
+                <div>
+                  <Typography variant="h3" gutterBottom>
+                    {institutionEditors.length > 0
+                      ? t('editor.institution.institution_editor_plural')
+                      : t('editor.institution.institution_editor_single')}
```

Comment:
Tror dette skal funke i sammenheng med endringsforslag i nbTranslations. Litt usikker på hvordan denne funker når man ikke eksplisitt setter en variabel. Mulig man må? 

```suggestion
                    {t('editor.institution.institution_editor', institutionEditors.length)}
```

Reply:
Liten oppdatering på forslaget over. Måtte legge til count, så funker det 👍

Reply:
Eneste som er kjipt med dette er om det kke er noen administrator/redaktør, for sa vil det defaule til "other", eller?

Reply:
Fortsatt bedre enn kun "...(er)" da, imo

Reply:
Ikke helt kjent med denne magien, var Kjetil som viste meg. Ser ut som det finnes en zero-ending man kan bruke i translation-fila.
https://www.i18next.com/translation-function/plurals

Reply:
JA, så på det. Men da blir det bare en duplikat, så vet ikke hvor mye vi skal legge i det. Er jo ikke ofte en institusjon ikke har noen administrator

Reply:
Ok. Blir ikke feil å defaulte til other når det mangler uansett, så da lar vi det være :)